OneSignalExample/Packages/com.onesignal.unity.core/Editor/SettingsWindow/SettingsTab.cs

```diff
@@ -0,0 +1,19 @@
+using Com.OneSignal.Editor.IMGUI;
+using UnityEditor;
+using UnityEngine;
+
+namespace Com.OneSignal.Editor
+{
+    class SettingsTab : WindowTabElement
+    {
+        public override void OnGUI()
+        {
+            var settings = OneSignalSettings.Instance;
+            using (new WindowBlockWithIndent(new GUIContent("Settings")))
+            {
+                EditorGUILayout.HelpBox("When Application Id is provided here, you may skip an appID param in the OneSignalPush.StartInit method.", MessageType.Info);
```

Comment:
@stan-osipov Thanks for adding the help box.
@jkasten2 Note to myself to update wording to include wording that OneSignal initialization code is still required to be added in the first Scene that start with your game.